pkgs/applications/editors/vscode/extensions/vscode-utils.nix

```diff
@@ -103,23 +103,23 @@ let
 
   toExtensionJsonEntry = ext: rec {
     identifier = {
-      id = ext.vscodeExtUniqueId;
+      id = ext.vscodeExtUniqueId or (lib.head (lib.attrNames (builtins.readDir "${ext}/share/vscode/extensions")));
```

Comment:
Is this using IFD?

Reply:
Only if `vscodeExtUniqueId` is missing, which right now throws an error.

Otherwise the behavoir is not changed.

Reply:
IFD is not allowed in nixpkgs and we should not use it as a fallback.

Reply:
The fix should probably to provide the `vscodeExtUniqueId` for lldb properly.

Reply:
Where?
```
# ~/src/nixpkgs on  vsc-ext-entry                                                                                     at 11:03:04 ❄ 
; rg '= "lldb'                                                                                                                # zsh 
pkgs/development/compilers/llvm/12/lldb/default.nix
27:  pname = "lldb";
118:  pname = "lldb-manpages";

pkgs/development/compilers/llvm/9/lldb/default.nix
22:  pname = "lldb";
33:      name = "lldb-use-system-debugserver-fix.patch";

pkgs/development/compilers/llvm/11/lldb/default.nix
21:  pname = "lldb";
102:  pname = "lldb-manpages";

pkgs/development/compilers/llvm/10/lldb/default.nix
21:  pname = "lldb";
104:  pname = "lldb-manpages";

pkgs/development/compilers/llvm/rocm/default.nix
450:    targetName = "lldb";

pkgs/development/compilers/llvm/15/lldb/default.nix
33:  pname = "lldb";
164:  pname = "lldb-manpages";

pkgs/development/compilers/llvm/5/lldb/default.nix
19:  pname = "lldb";

pkgs/development/compilers/llvm/git/lldb/default.nix
28:  pname = "lldb";
121:  pname = "lldb-manpages";

pkgs/development/compilers/llvm/14/lldb/default.nix
29:  pname = "lldb";
143:  pname = "lldb-manpages";

pkgs/development/compilers/llvm/13/lldb/default.nix
28:  pname = "lldb";
134:  pname = "lldb-manpages";

pkgs/development/compilers/llvm/8/lldb/default.nix
19:  pname = "lldb";

pkgs/development/compilers/llvm/7/lldb/default.nix
19:  pname = "lldb";

pkgs/development/compilers/llvm/6/lldb/default.nix
19:  pname = "lldb";
```

Lldb is clearly machine-generated, but I do not know where it is generated.